otter/cloud_client/__init__.py

```diff
@@ -501,6 +502,15 @@ class CLBNodeLimitError(Exception):
     """
 
 
+@attr.s
+class CLBPartialNodesRemoved(Exception):
+    """
+    Exception raised when some of the nodes are removed.
+    """
+    lb_id = attr.ib()
+    node_ids = attr.ib()    # Node ids not removed
```

Comment:
Actually upon reviewing the code later on, I'm pretty sure I might want (as a user of this as a library) to see both `removed_node_ids` and `not_removed_node_ids` as separate attributes here.